python/its-status/its_status/collector.cellular.py

```diff
@@ -8,8 +8,8 @@
 
 
 class Status:
-    def __init__(self, cfg):
```

Comment:
I'll fix all the collectors whch do not use the config.

Reply:
Arg, no, that is not good, not good, not good.

In fact, this is a **keyword-only** parameter, so it **must** be named `cfg`.
We can only underscore-prefix positional-only arguments.

And that last-minute change was not tested,neither before being committed
not before being merged, and that breaks at runtime. That is not good at all,
shame on me...

I got totally misled because your comment is on a removed line f code, not on
the new one. :-(

Reply:
Fixed with #65.